nomos-core/src/block/builder.rs

```diff
@@ -0,0 +1,100 @@
+// std
+use serde::de::DeserializeOwned;
+use serde::Serialize;
+use std::hash::Hash;
+use std::marker::PhantomData;
+// crates
+
+// internal
+use crate::block::Block;
+use crate::da::blob::{Blob, BlobSelect};
+use crate::tx::{Transaction, TxSelect};
+use consensus_engine::overlay::RandomBeaconState;
+use consensus_engine::{NodeId, Qc, View};
+
+pub struct BlockBuilder<Tx, Blob, TxSelector, BlobSelector> {
+    _tx: PhantomData<Tx>,
+    _blob: PhantomData<Blob>,
```

Comment:
It seems not need `PhantomData<_>` here as we have `Item = ...`.

Reply:
Oh yes, this is a leftover from early writing, thanks!